pkg/Microsoft.Windows.Compatibility/tests/publishcompatibilityassets.ps1

```diff
@@ -43,9 +44,9 @@ if (!(Test-Path $localPackageSourcePath))
 $compatPackageVersion = _getPackageVersion "Microsoft.Windows.Compatibility"
 $privatePackageVersion = _getPackageVersion "Microsoft.Private.CoreFx.NETCoreApp"
 
-& $dotnetPath restore --packages $packagesCachePath /p:RestoreSources="$restoreSources" /p:TargetFramework=$targetFramework /p:CompatibilityPackageVersion=$compatPackageVersion /p:PrivateCorefxPackageVersion=$privatePackageVersion /p:RID=$rid $csprojPath
+& $dotnetPath restore --packages $packagesCachePath /p:RestoreSources="$restoreSources" /p:TargetFramework=$targetFramework /p:CompatibilityPackageVersion=$compatPackageVersion /p:PrivateCorefxPackageVersion=$privatePackageVersion /p:RID=$rid /p:NETCoreAppMaximumVersion=$netcoreappVersion $csprojPath
```

Comment:
I suspect you can just set this in the project as well to avoid needing to pass it everywhere.

Reply:
Good idea.

Reply:
Fixed.